scripts/openstack-update.sh

```diff
@@ -115,7 +115,7 @@ oc wait $OPENSTACK_VERSION_CR --for=condition=MinorUpdateOVNControlplane --timeo
 get_current_state "02_after_ovn_controlplane_update"
 
 # start ovn update on data plane
-DATAPLANE_NODESET=$(oc get openstackdataplanenodeset -o name | awk -F'/' '{print "    - "  $2}')
+DATAPLANE_NODESET=$(oc get openstackdataplanenodeset -o name | awk -F'/' '/openstack-edpm/ {print "    - "  $2}')
```

Comment:
In my mind more robust solution would be checking service list of openstackdataplanenodeset to verify if own service is on the list. If ovn is on list, run ovn update, else skip the own update.